src/service/ui/messaging.js

```diff
@@ -833,7 +833,10 @@ export const Window = GObject.registerClass({
 }, class MessagingWindow extends Gtk.ApplicationWindow {
 
     _init(params) {
-        super._init(params);
+        super._init(Object.assign({
+            application: Gio.Application.get_default(),
+            title: _('Messaging'),
```

Comment:
This is already set in the template, setting it here is redundant and makes maintenance harder:

https://github.com/GSConnect/gnome-shell-extension-gsconnect/blob/3758807fac3f2b3c09a30a1767cf7f551d27d915/data/ui/messaging-window.ui#L21